pkg/shell/index.js

```diff
@@ -378,7 +490,6 @@ define([
             label = _("Machines");
         }
         $("#machine-link span").text(label);
-        $("title").text(title || default_title);
```

Comment:
I think this was the only use of title. Since it's removed should we remove the declaration and assignment above as well?